index.php

```diff
@@ -164,3 +166,23 @@
 	header('Content-Type: text/html; charset=utf-8');
 	print $hyphaHtml->toString();
 	exit;
+
+  /* Function:
+	* createSearch
+	* makes the html for the search bar
+	*/
+function createSearch()
+{
+  $search = __('search');
+	$searchbar= <<<EOF
+	<!-- HTML for SEARCH BAR -->
+	<span >
+	<input id="mysearchPattern" type="text" class="searchinput" placeholder="$search ..." name="q" size="21" maxlength="120" style="width: 300px; ">
+	<button style="margin-left: 15px; margin-right: 15px;" class="searchbutton" type="search"
```

Comment:
Should this styling be here, or in the CSS file? Same for styling on input field above.